core/cmd/local_client.go

```diff
@@ -251,14 +251,14 @@ func ensureKeys(ctx context.Context, cfg config.GeneralConfig, ethClient eth.Cli
 		return errors.Wrap(err, "failed to ensure ocr key")
 	}
 	if !didExist {
```

Comment:
Would we want to respect the FeatureOffchainReporting flag here still? i.e. not create OCR keys if thats unset

Reply:
We didn't previously, so I didn't introduce that optimization. Seems harmless enough to just create the keys anyway.